guides/micronaut-cloud-database-base/create-app.adoc

```diff
@@ -0,0 +1,3 @@
+== Creating the Application
```

Comment:
Could not we just have this in a common snippet name `micronaut-cloud-database-create-app`?